controllers/app.controller.js

```diff
@@ -50,10 +50,9 @@ exports.getCommentsById = (req, res, next) => {
 exports.postCommentById = (req, res, next) => {
 	const id = req.params.article_id;
 	const comment = req.body;
-	const promises = [checkExists(id), insertCommentById(id, comment)];
-	Promise.all(promises)
-		.then(() => {
-			res.status(201).send({ comment });
+	insertCommentById(id, comment)
```

Comment:
This is a small thing, but I'd assume the id in this name is the `comment_id` from the name of this function, could you name this function to make it clearer it's using an `article_id`? Same with `postCommentById`